node/agent_proxy.go

```diff
@@ -158,55 +123,10 @@ func (a *HostAgent) AckProxySnapshotQuiece(snapshotId string, unused *interface{
 	return errors.New("unimplemented")
 }
 
-// GetHealthCheck returns the health check configuration for a service, if it exists
-func (a *HostAgent) GetHealthCheck(req HealthCheckRequest, healthChecks *map[string]health.HealthCheck) error {
-	glog.V(4).Infof("ControlCenterAgent.GetHealthCheck()")
-	*healthChecks = make(map[string]health.HealthCheck, 0)
-
-	controlClient, err := NewControlClient(a.master)
-	if err != nil {
-		glog.Errorf("Could not start Control Center client %v", err)
-		return err
-	}
-	defer controlClient.Close()
-
-	var svc service.Service
-	err = controlClient.GetService(req.ServiceID, &svc)
-	if err != nil {
-		return err
-	}
-	getSvc := func(svcID string) (service.Service, error) {
-		svc := service.Service{}
-		err := controlClient.GetService(svcID, &svc)
-		return svc, err
-	}
-
-	findChild := func(svcID, childName string) (service.Service, error) {
-		svc := service.Service{}
-		err := controlClient.FindChildService(dao.FindChildRequest{svcID, childName}, &svc)
-		return svc, err
-	}
-	svc.EvaluateHealthCheckTemplate(getSvc, findChild, req.InstanceID)
-	if svc.HealthChecks != nil {
-		*healthChecks = svc.HealthChecks
-	}
-	return nil
-}
-
-// LogHealthCheck proxies RegisterHealthCheck.
-func (a *HostAgent) LogHealthCheck(result domain.HealthCheckResult, unused *int) error {
-	controlClient, err := NewControlClient(a.master)
-	if err != nil {
-		glog.Errorf("Could not start Control Center client %v", err)
-		return err
-	}
-	defer controlClient.Close()
-	err = controlClient.LogHealthCheck(result, unused)
-	return err
-}
 
 // ReportHealthStatus proxies ReportHealthStatus to the master server.
 func (a *HostAgent) ReportHealthStatus(req dao.HealthStatusRequest, unused *int) error {
+	// FIXME: use new master.ClientInterface instead
```

Comment:
Yes - I'm in the process of moving all of `dao/elasticsearch/health.go` stuff to `rpc`